app/src/main/res/layout/fragment_scheduler.xml

```diff
@@ -8,189 +8,272 @@
-                    bind:topSheetLayout="@{layoutTopSheet}"
-                    bind:viewModel="@{viewModel}"
-                    />
-
-            </LinearLayout>
-
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="?attr/actionBarSize"
-                android:background="@color/colorPrimary"
-                android:gravity="center_vertical"
-                android:orientation="horizontal"
-                >
+                <android.support.v7.widget.RecyclerView
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="@color/color_grey_100"
+                    bind:layoutManager="@{viewModel.layoutManager}"
+                    bind:recyclerAdapter="@{viewModel.adapter}"/>
 
-                <TextView
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginLeft="@dimen/dp_20"
-                    android:layout_marginStart="@dimen/dp_20"
-                    android:layout_weight="1"
-                    android:text="@{viewModel.titleTopSheet}"
-                    android:textColor="@color/color_white"
-                    android:textSize="@dimen/sp_20"
-                    android:textStyle="bold"/>
-
-                <Spinner
+                <ProgressBar
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginEnd="@dimen/dp_20"
-                    android:layout_marginRight="@dimen/dp_20"
-                    bind:array="@{@stringArray/title_status}"
-                    bind:listener="@{viewModel.spinnerSelectListener}"/>
+                    android:layout_gravity="bottom|center"
+                    android:layout_marginBottom="@dimen/dp_16"
+                    android:visibility="@{viewModel.loadingMore?View.VISIBLE:View.GONE}"/>
 
-                <ImageView
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginEnd="@dimen/dp_20"
-                    android:layout_marginRight="@dimen/dp_20"
-                    android:onClick="@{()->viewModel.onClickTopSheet(layoutTopSheet)}"
-                    app:srcCompat="@drawable/ic_swipe_down_24dp"/>
-            </LinearLayout>
-        </LinearLayout>
+                <com.github.clans.fab.FloatingActionMenu
+                    android:id="@+id/menu_yellow"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="bottom|left"
+                    android:paddingBottom="@dimen/dp_10"
+                    android:paddingLeft="@dimen/dp_16"
+                    android:paddingStart="@dimen/dp_16"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.